addons/a11y/src/components/A11YPanel.tsx

```diff
@@ -20,26 +20,29 @@ export enum RuleType {
   INCOMPLETION,
 }
 
-const RotatingIcons = styled(Icons)(({ theme }) => ({
+const Icon = styled(Icons)({
```

Comment:
@donaldpipowitch  I needed to keep an `Icon` component with `height` and `width` otherwise the check icon from https://github.com/storybookjs/storybook/pull/8883/commits/e368e570989a85b7f8cb7565ecced624134ee4d2#diff-3250ce08c136d40dd47659131333a6a7R213 is not displayed